es-design-system/pages/atoms/icons.vue

```diff
@@ -481,6 +477,20 @@
                 <code>IconFilePdf</code>
             </li>
         </ul>
+        <h2>
+            Miscellaneous Icons
+        </h2>
+        <p>
+            Default size for file icons is 74px by 54px.
```

Comment:
The icons in this section will probably not end up being consistent sizes - we can probably remove this note so we don't need to figure out how to update it when we add a new one.